bindgen/src/utils.ts

```diff
@@ -76,3 +76,9 @@ export function posixRelativePath(from: string, to: string): string {
 export function toString(node: Node): string {
   return ASTBuilder.build(node);
 }
+
+const captialPattern = /([A-Z])/g;
+
+export function make_snake_case(s: string): string {
```

Comment:
this should probably be camel case to fit the rest of the functions:

```suggestion
export function makeSnakeCase(s: string): string {
```

Reply:
Lol fine, I meant it as a joke, but we should maintain a single style.